pycaret/internal/tabular.py

```diff
@@ -4,6 +4,7 @@
 # Release: PyCaret 2.2
 # Last modified : 26/08/2020
 
+from cgi import test
```

Comment:
Is this import needed?

Reply:
I did not add this particular line. I simply fetched the latest push in the main repo and added my code, I have not added any import in any file.